great_expectations/rule_based_profiler/parameter_builder/regex_pattern_string_parameter_builder.py

```diff
@@ -234,10 +234,8 @@ def _build_parameters(
             parameters=parameters,
         )
 
-        # get list of regex_strings that match greater than threshold
-        regex_string_success_list: List[
-            str
-        ] = self._get_regex_matched_greater_than_threshold(
+        # get best-matching regex_string that match greater than threshold
+        best_regex_string: str = self._get_best_regex_greater_than_threshold(
             regex_string_success_ratios, threshold
         )
```

Comment:
@alexsherstinsky the regex parameter builder has been modified to output the best regex-string, so now the behavior is consistent with the datetime parameter builder :)